indexer_app/tasks.py

```
@@ -25,35 +36,61 @@ async def indexer(from_block: int, to_block: int):
+            logger.info(
+                f"Time to fetch new block: {fetch_end_time - fetch_start_time:.4f} seconds"
+            )
+            # Log time before caching block height
+            save_start_time = time.time()
+            # Update current block height
+            asyncio.create_task(
+                save_block_height(
+                    streamer_message.block.header.height,
+                    streamer_message.block.header.timestamp,
+                )
+            )
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)